development.md

```diff
@@ -0,0 +1,68 @@
+# Local Dev Guide
+
+These are in progress development iteration steps to run the following:
+
+* a local `kind` cluster
+* Crossplane as a helm chart from source
+* `provider-aws` and `provider-helm` from published packages
+* directly apply XRDs and Composition manifests with `kubectl`
+
+## kind cluster and Crossplane
+
+```console
+cluster/local/kind.sh up
```

Comment:
Might be worth mentioning here that we need to be in `crossplane/crossplane` because we also have a `cluster/` dir in this repo

Reply:
good idea, will add!